# HG changeset patch
# User Olaf Hering <olaf@xxxxxxxxx>
# Date 1354786108 -3600
# Node ID 5dab922f04f4790b70d43d062114bddcc1bb7de6
# Parent  670b07e8d7382229639af0d1df30071e6c1ebb19
tools/gdbsx: fix build failure with glibc-2.17

[ 1029s] xg_main.c: In function '_domctl_hcall':
[ 1029s] xg_main.c:181:52: error: 'ulong' undeclared (first use in this 
function)
[ 1029s] xg_main.c:181:52: note: each undeclared identifier is reported only 
once for each function it appears in
[ 1029s] xg_main.c: In function '_check_hyp':
[ 1029s] xg_main.c:221:52: error: 'ulong' undeclared (first use in this 
function)
[ 1029s] make[4]: *** [xg_main.o] Error 1

Signed-off-by: Olaf Hering <olaf@xxxxxxxxx>

diff -r 670b07e8d738 -r 5dab922f04f4 tools/debugger/gdbsx/xg/xg_main.c
--- a/tools/debugger/gdbsx/xg/xg_main.c
+++ b/tools/debugger/gdbsx/xg/xg_main.c
@@ -34,6 +34,7 @@
  *  XGTRC(): generic trace utility
  */
 
+#include <sys/types.h>
 #include <stdio.h>
 #include <stddef.h>
 #include <stdarg.h>

